src/Idris/REPL.idr

```diff
@@ -800,6 +800,9 @@ process (Editing cmd)
 process (CGDirective str)
     = do setSession (record { directives $= (str::) } !getSession)
          pure Done
+process (RunShellCommand cmd)
+    = do coreLift (system cmd)
```

Comment:
Should we check we are not getting back an error code?
At least to warn the user?

Reply:
@gallais I don't think so. Bash does not by default. Neither does Idris 1. Most commands print their error messages when they error. Is there a specific use case you had in mind?

Reply:
> Is there a specific use case you had in mind?

Nope. I just wanted to make sure it was not an oversight.